backend/models/fileRef.js

```diff
@@ -251,17 +269,22 @@ FileRef.storeMitigationsFile = async function(account, user, name, data) {
 	return await this.storeFile(account, collName, user, name, data, extraFields);
 };
 
-FileRef.storeFileAsResource = async function(account, model, user, name, data, extraFields = null) {
-	const collName = model + RESOURCES_FILE_REF_EXT;
+FileRef.storeFile = async function(account, model, collName, data, extraFields = null) {
```

Comment:
these changes looks like it's reverted the refactoring effort done in safetibase? I don't know if it's because it's missing a merge with the latest code